client/app/dedicatedUniverseComponents/price/price.filter.js

```diff
@@ -40,16 +50,18 @@ export default /* @ngInject */ ($translate) => {
     NL: frenchTouch,
     PL: frenchTouch,
     PT: frenchTouch,
-    SG: usTouch,
+    SG: asiaTouch,
     TN: frenchTouch,
     US: usTouch,
   };
```

Comment:
Does it make sense to handle this has capabilities instead ? I had in mind something like 
```suggestion
 shouldUseGst(subs) {
  ['ASIA', 'AU'].includes(subs)
 }
```

Reply:
I dont think this would work. These objects are more like configuration. This might be a good use case to fetch templates for respective taxation type.